clients/routes.ts

```diff
@@ -174,6 +174,21 @@ const routes = {
       },
     },
   },
+  inpi: {
+    api: {
+      rne: {
+        login: 'https://registre-national-entreprises.inpi.fr/api/sso/login',
+        cmc: {
```

Comment:
I feel like it would be clearer to use actes/documents variables names. 

maybe 
```
inpi.api.rne.login
inpi.api.rne.actes
inpi.api.rne.actes.list
inpi.api.rne.actes.download
```